config/environment.js

```diff
@@ -54,7 +56,13 @@ module.exports = function (environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
-    ENV.featureFlags.simpleLogin = true;
+    const simpleLogin = process.env.SIMPLE_LOGIN;
```

Comment:
unfortunately this will only be read on build, so won't work for our containers. There is an alternative solution here: https://github.com/mu-semtech/ember-proxy-service/#using-environment-variables-at-run-time-with-an-ember-application but that only works for strings.